src/Lib.hs

```diff
@@ -15,7 +15,7 @@ hardIO space time = do
     timer <- spin time
     arrTotal <- sum <$> mapM (readArray arr) [1..space]
     random <- fst . next <$> newStdGen
-    return $ arrTotal + random
+    return $! arrTotal + random
```

Comment:
this forces the thunk to be evaluated to WHNF (which in the case of an Int is fully evaluated)